src/Caching/Cache.php

```diff
@@ -98,6 +98,45 @@ public function load($key, $fallback = NULL)
 
 
 	/**
+	 * Reads multiple items from the cache
+	 * @param array
+	 * @param callable
+	 * @return array
+	 */
+	public function bulkLoad(array $keys, $fallback = NULL)
```

Comment:
Does it make sense to have one fallback? Woulldn't make more sense to have fallback for each key? Or support both?